modules/aggregations/src/yamlRestTest/resources/rest-api-spec/test/aggregations/median_absolute_deviation_metric.yml

```diff
@@ -76,6 +76,26 @@ setup:
   - match: { aggregations.mad_int.value: 100 }
   - match: { aggregations.mad_double.value: 100 }
 
+---
+"with execution hint":
+
+  - skip:
+      version: " - 8.8.99"
+      reason: t-digest forked in 8.9.0
+  - do:
+      search:
+        rest_total_hits_as_int: true
```

Comment:
I think this is supported but we tend not to use `rest_total_hits_as_int ` any more in our tests...probably at some point we will get rid of it.

Reply:
Removed in my added test. Agreed, we probably need a cleanup PR just for this.